recipes/mosh/build.sh

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/conda-forge/toolchain-feedstock/pull/11
+export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include $(pkg-config --cflags zlib)"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib $(pkg-config --libs zlib)"
```

Comment:
You should use either `-L${PREFIX}/lib` or `$(pkg-config --libs-only-L zlib)`. Both are equivalent.
`$(pkg-config --libs zlib)` on the other hand adds `-lz` as well, which is unnecessary to find the libraries.